src/lib.rs

```diff
@@ -179,6 +179,7 @@ mod extensions;
 pub use convert::HttpTryFrom;
 pub use error::{Error, Result};
 pub use extensions::Extensions;
+#[doc(no_inline)]
```

Comment:
Why preventing the inline here? I mostly just don't know what this does...

Reply:
More about it https://doc.rust-lang.org/rustdoc/the-doc-attribute.html#docno_inlinedocinline